crates/storage-pg/migrations/20241212154426_oauth2_response_mode_null.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE "upstream_oauth_providers" ALTER COLUMN "response_mode" DROP NOT NULL;
```

Comment:
Can you add the copyright header + a short description of what this migration does?